src/main/resources/api.yaml

```diff
@@ -131,7 +131,7 @@ paths:
         - application/json
       parameters:
         - in: path
-          name: projectId
+          name: id
```

Comment:
Tu się chyba trzymamy strategii z bardziej verbose nazwą - na masterze to jest poprawione - rebase?